tools/syz-testbed/stats.go

```diff
@@ -417,7 +417,19 @@ func (group *RunResultGroup) SaveAvgBenchFile(fileName string) error {
 		return err
 	}
 	defer f.Close()
-	for _, averaged := range group.AvgStatRecords() {
+	// In long runs, we collect a lot of stat samples, which results
+	// in large and slow to load graphs. A subset of 256-512 data points
+	// seems to be a reasonable enough precision.
+	records := group.AvgStatRecords()
+	const targetRecords = 256
+	for len(records) > targetRecords*2 {
+		newRecords := make([]map[string]uint64, 0, targetRecords)
```

Comment:
Do they grow by 1? If not then, it's better to use `len(records)/2` instead of `targetRecords`.

Reply:
Fixed, PTAL.

I also reduced the number of points to 128-256, it seems to be enough.